extensions/sql-database-projects/src/dialogs/addSqlBindingQuickpick.ts

```diff
@@ -204,7 +215,26 @@ export async function launchAddSqlBindingQuickpick(uri: vscode.Uri | undefined,
 					}
 					if (connectionString) {
 						try {
-							const success = await azureFunctionsUtils.setLocalAppSetting(path.dirname(projectUri.fsPath), newConnectionStringSettingName, connectionString);
+							const projectFolder: string = path.dirname(projectUri.fsPath);
+							const localSettingsPath: string = path.join(projectFolder, constants.azureFunctionLocalSettingsFileName);
+							// Connection string does not include a password, so will prompt user to optionally enter it manually
+							if (connectionString.includes(`Password=${constants.passwordPlaceholder}`)) {
```

Comment:
Would be better to just take the answer the user gave above and use that instead of relying on this value never changing.